packages/react/src/components/fileInput/FileInput.test.tsx

```diff
@@ -0,0 +1,257 @@
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { axe } from 'jest-axe';
+
+import { FileInput } from './FileInput';
+
+// eslint-disable-next-line
+const onChangeTest = () => {};
+
+describe('<FileInput /> spec', () => {
+  it('renders the component', () => {
+    const { asFragment } = render(
+      <FileInput
+        id="test-file-input"
+        label="Choose a file"
+        buttonLabel="Add file"
+        language="en"
+        accept=".png,.jpg"
+        onChange={onChangeTest}
+      />,
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should not have basic accessibility issues', async () => {
```

Comment:
Would it be good idea to also test the accessibility in a state where there are multiple files added?

Reply:
It could. I need to figure out how I'm going to do it 👍

Reply:
- [ ] add accessibility test for state where files are selected files